lib/DocumentAccessor.php

```diff
@@ -61,6 +61,15 @@ public function set($field, $value)
         $property->setValue($this->document, $value);
     }
 
+    public function get($field)
+    {
+        $property = $this->reflection->getProperty($field);
+        // TODO: Can be cached? Makes a performance diff?
+        $property->setAccessible(true);
```

Comment:
Should use the Metadata class in the future to set private properties, in this way the reflection class and properties will only instantiated one time per document class per request.